### ISSUE TITLE
chess.svg.MARGIN not there

### ISSUE THREAD

**Issue by PedanticHacker** [niklasf/python-chess#317]
After installing the latest version of python-chess (i.e., **v0.23.10)**, I tried using the new `MARGIN` constant from the `chess.svg` module. Well, it apparently isn't there? What gives?

```
self.margin = chess.svg.MARGIN if COORDINATES else 0
AttributeError: module 'chess.svg' has no attribute 'MARGIN'
```

**Comment by niklasf** [comment]
The 0.23.x branch is now maintained with bugfixes only. All new features are developed on the master branch (which will soon have the first Python 3 only release).

**Comment by PedanticHacker** [comment]
Ah, okay. Got any timeline when this _Python 3 only_ release will occur?

**Comment by niklasf** [comment]
Definitely 2018. If you need it right now, you could use pip to install python-chess [from Git](https://pip.pypa.io/en/stable/reference/pip_install/#git).

**Comment by PedanticHacker** [comment]
I noticed that there's no difference between setting the margin to 20 or to 40 in my application. It works exactly the same. How is that possible?

Also, I noticed that clicking on a square of the SVG chessboard does not select that square (but rather the neighboring one) if the click is not done in the center of it. Does that have to do anything with DPI scaling of Windows 10? My DPI scaling setting on Windows 10 is set to the recommended 150%. I think python-chess' SVG chessboard does not take any DPI scaling into account, or does it?
